File: src/main/webapp/pages/app/services/peminjamanService.js
"use strict";

angular.module('perpustakaan')
  .factory('PeminjamanService', ['$http', function($http) {
    return {
      getAllPeminjaman: function() {
        return $http.get('/api/peminjaman');
      },
      deletePeminjaman: function(idPeminjaman) {
        return $http.delete('/api/peminjaman/' + idPeminjaman);
      }
    };

  }]);

File: src/main/java/com/perpustakaan/markaz/syariah/service/PeminjamanService.java
package com.perpustakaan.markaz.syariah.service;

import com.perpustakaan.markaz.syariah.domain.Peminjaman;

import java.util.List;

/**
 * Created by rizki on 27/06/15.
 */
public interface PeminjamanService {
    void save(Peminjaman peminjaman, String noKode);
    void delete(String idPeminjaman);
    Peminjaman getPeminjaman(String idPeminjaman);
    List<Peminjaman> getPeminjamans();
}
